client/verta/requirements.txt

```
@@ -2,6 +2,7 @@ click >= 7.0, < 9.0
 cloudpickle >= 1.0, < 3.0
 googleapis-common-protos >= 1.5, < 2.0
 importlib-metadata >= 3.7.0
+jsonschema>=4.17.3
```

We should give it some space (not actually required, but makes it slightly slightly easier to read)

```suggestion
jsonschema >= 4.17.3
```